controller/eks-cluster-config-handler.go

```diff
@@ -335,7 +336,7 @@ func validateUpdate(config *eksv1.EKSClusterConfig) error {
 		if _, ok := nodeGroupNames[aws.ToString(ng.NodegroupName)]; !ok {
 			nodeGroupNames[aws.ToString(ng.NodegroupName)] = struct{}{}
 		} else {
-			errs = append(errs, fmt.Sprintf("node group names must be unique within the [%s] cluster to avoid duplication", config.Name))
+			errs = append(errs, fmt.Sprintf("node group name %q must be unique within the [%s (id: %s)] cluster to avoid duplication", aws.ToString(ng.NodegroupName), config.Spec.DisplayName, config.Name))
```

Comment:
```suggestion
			errs = append(errs, fmt.Sprintf("node group names must be unique within the [%s (id: %s)] cluster to avoid duplication", config.Spec.DisplayName, config.Name))
```